test/setup.sh

```diff
@@ -94,25 +85,14 @@ die() {
 
 trap 'die' HUP INT QUIT ABRT TERM;
 
-
-
 # Database setup
 cleanup;
 create_db;
 
-# echo -e "\nInstalling cartodb extension";
+echo -e "\nInstalling cartodb extension";
```

Comment:
Please delete this command, it was used before calling `CREATE EXTENSION cartodb` but that was removed before.